Common/Data/Custom/Benzinga/BenzingaNewsJsonConverter.cs

```diff
@@ -0,0 +1,283 @@
+            token.WriteTo(writer);
+        }
+
+        /// <summary>
+        /// Reads the JSON representation of the object.
+        /// </summary>
+        /// <param name="reader">The <see cref="T:Newtonsoft.Json.JsonReader"/> to read from.</param><param name="objectType">Type of the object.</param><param name="existingValue">The existing value of object being read.</param><param name="serializer">The calling serializer.</param>
+        /// <returns>
+        /// The object value.
+        /// </returns>
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            var data = JToken.Load(reader);
+            var instance = DeserializeNews(data);
+
+            instance.Symbol = _symbol;
+            instance.EndTime = instance.UpdatedAt;
+
+            return instance;
+        }
+
+        /// <summary>
+        /// Determines whether this instance can convert the specified object type.
+        /// </summary>
+        /// <param name="objectType">Type of the object.</param>
+        /// <returns>
+        /// <c>true</c> if this instance can convert the specified object type; otherwise, <c>false</c>.
+        /// </returns>
+        public override bool CanConvert(Type objectType)
+        {
+            return objectType == typeof(BenzingaNews);
+        }
+
+        /// <summary>
+        /// Helper method to deserialize a single json Benzinga news
+        /// </summary>
+        /// <param name="item">The json token containing the Benzinga news to deserialize</param>
+        /// <param name="enableLogging">true to enable logging (for debug purposes)</param>
+        /// <returns>The deserialized <see cref="BenzingaNews"/> instance</returns>
+        public static BenzingaNews DeserializeNews(JToken item, bool enableLogging = false)
+        {
+            var instance = new BenzingaNews
+            {
+                Id = item.Value<int>("id"),
+                Author = item.Value<string>("author"),
+                CreatedAt = item.Value<DateTime>("created").ToUniversalTime(),
+                UpdatedAt = item.Value<DateTime>("updated").ToUniversalTime(),
+                Title = item.Value<string>("title"),
+                // Teasers are not present in the RSS data
+                Teaser = item.Value<string>("teaser"),
+                // Strip all HTML tags from the article, then convert HTML entities to their string representation
+                // e.g. "<html><p>Apple&#39;s Earnings</p></html>" would become "Apple's Earnings"
+                Contents = WebUtility.HtmlDecode(Regex.Replace(item.Value<string>("body"), @"<[^>]*>", " ")),
+                Categories = new List<string>(),
+                Symbols = new List<Symbol>(),
+                Tags = new List<string>(),
+            };
+
+            instance.Contents = WebUtility.HtmlDecode(Regex.Replace(instance.Contents, @"<[^>]*>", " "));
```

Comment:
I think `WebUtility.HtmlDecode(Regex.Replace` is already being done in the initialization?